### PR TITLE
feat(alias extraction): Add client_attrs (Client Attributes) docs

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

**Review by zmstone — DISMISSED** [review]

File: en_US/configuration/configuration.md [hunk 1]
```diff
@@ -393,6 +393,146 @@ will be interpreted as `myarray = [74, 75]`, which is handy when trying to overr
 
 :::
 
+### Variform Expressions
+
+Variform is a lightweight, expressive language designed for string manipulation and runtime evaluation.
+It is not a full-fledged programming language but rather a specialized tool that can be embedded within
+configurations or EMQX to perform string operations dynamically.
```
Comment (zmstone) [inline en_US/configuration/configuration.md]: or EMQX -> for EMQX

File: en_US/client-attributes/client-attributes.md [hunk 2]
```diff
@@ -1,98 +1,161 @@
...
+- `dn`: DN field of the TLS certificate
+
+- `user_property.*`: Extracts attribute values from User-Property in the MQTT CONNECT packet, e.g., `user_property.foo`
 
-- `cn`: Client certificate common name
-- `dn`: Client certificate distinguish name (Subject)
-- `clientid`
-- `username`
-- `user_property`: The user properties provided in the MQTT v5 CONNECT packet sent by the client
 - `ip_address`: The source IP of the client
```
Comment (wivwiv) [inline en_US/client-attributes/client-attributes.md]: @zmstone It looks like ip_address and port are not supported. 
```bash
2024-05-16T07:58:50.980120+00:00 [warning] msg: client_attr_initialization_failed, peername: 127.0.0.1:57746, reason: #{reason => var_unbound,var_name => "ip_address"}
2024-05-16T07:58:50.983633+00:00 [warning] msg: client_attr_initialization_failed, peername: 127.0.0.1:57746, reason: #{reason => var_unbound,var_name => "port"}
```

Comment (zmstone) [reply en_US/client-attributes/client-attributes.md]: my bad, it's a part of the connection property, but not client property.
we can delete this from the docs for now?

File: en_US/access-control/authn/http.md [hunk 3]
```diff
@@ -26,10 +27,18 @@ Headers: Content-Type: application/json
...
+    "client_attrs": {
+        "role": "admin",
+        "sn": "10c61f1a1f47"
+    }
 }
 ```
 
+
+
+`client_attrs` 是可选的
```
Comment (wivwiv) [inline en_US/access-control/authn/http.md]: typo

File: en_US/client-attributes/client-attributes.md (hunk 2 above)
Comment (wivwiv) [reply en_US/client-attributes/client-attributes.md]: @Meggielqk please delete them

File: zh_CN/access-control/authz/authz.md [hunk 4]
```diff
@@ -111,6 +113,7 @@ EMQX 还允许在主题中使用占位符，在匹配规则时将当前客户端
 
 - `${clientid}`
 - `${username}`
+- `${client_attrs.NAME}`: 某一个客户端属性，`NAME` 替换成 `mqtt.client_attrs_init` 规则提取的某一个字段名称。
```
Comment (wivwiv) [inline zh_CN/access-control/authz/authz.md:116]:
```suggestion
- `${client_attrs.NAME}`: 某一个客户端属性，`NAME` 替换成 [客户端属性](../../client_attributes/client_attributes.md)设置的属性名称。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by zmstone — APPROVED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**